app/(protected)/albums/[id]/page.tsx

```
@@ -0,0 +1,135 @@
+				{isMenuOpen && (
+					<motion.div
+						initial={{ opacity: 0, y: -20 }}
+						animate={{ opacity: 1, y: 0 }}
+						exit={{ opacity: 0, y: -20 }}
+						transition={{ duration: 0.2 }}
+					>
+						<div>
+							<button onClick={() => setEditMode("coverPhoto")}>
+								カバー写真を変更
+							</button>
+							<button onClick={() => setEditMode("title")}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)